### ISSUE TITLE
App Builder parameters not parsed correctly

### ISSUE THREAD

**Issue by bjhardcastle** [AllenNeuralDynamics/aind-capsule-ephys-spikesort-kilosort25-full#20]
Trying to set `max_channel_fraction: 0.95`, the start of `output` shows that the default `0.5` is still being used:
```
python -u run_capsule_aind.py true false cmr false true 0.95 30
Running processing with the following parameters:
	CONCATENATE: False
	DENOISING_STRATEGY: cmr
	REMOVE_OUT_CHANNELS: False
	REMOVE_BAD_CHANNELS: True
	MAX BAD CHANNEL FRACTION: 0.5
	N_JOBS: -1
```

It might be necessary to switch to named parameters now that there are optional parameters in `ArgumentParser`.

**Comment by alejoe91** [comment]
Thanks, I'll check! The argoarse parsing might be wrong

**Comment by alejoe91** [comment]
Fixed here: 542279819b3bf2e5f97c598659ddcc1fe44950fc

The problem was that the named parameter was the first entru of the `or`, and having a default value it would always shadow the positional arg
